build.gradle

```diff
@@ -38,14 +38,10 @@ subprojects {
     configurations.all {
         resolutionStrategy {
             failOnVersionConflict()
-            // Force the IDEA SDK provided version of these dependencies to avoid classloading issues.
-            force 'org.apache.httpcomponents:httpclient:4.4.1', 'com.google.guava:guava:17.0'
```

Comment:
it looks like it would have to be something like this inside the `resolutionStrategy` block, but this feels weird... :             
```
eachDependency { DependencyResolveDetails dependency ->
  if (dependency.requested.group == "org.apache.httpcomponents" && dependency.requested.name == "httpclient" ||
      dependency.requested.group == "com.google.guava" && dependency.requested.name == "guava") {
    throw new GradleException('Dependency should not be included in your build')
  }
}
```